pyzo/pyzokernel/introspection.py

```diff
@@ -254,6 +254,7 @@ def storeInfo(name, val):
                         hasattr(val, "__array__")
                         and hasattr(val, "dtype")
                         and hasattr(val, "shape")
+                        and not hasattr(val, "__this_object_is_definitely_not_an_array__")
```

Comment:
A small tweak so that a future reader understand why this case is here. I actually looked up `__this_object_is_definitely_not_an_array__` on Google because my initial thought was that this is a convention 😆 
```suggestion
                        and not hasattr(val, "if_this_is_an_attribute_then_there_are_likely_inf_attributes")
```